### ISSUE TITLE
Special characters not supported in ZITI_PWD

### ISSUE THREAD

**Issue by gberl002** [openziti/ziti#1217]
When using a special character as the password for the quickstart, either through setting `ZITI_PWD` explicitly or entering a custom password during the quickstart causes future issues.

The password is fine as-is in the environment variable but is stored in the .env file with additional single quotes around it. So, when sourcing the .env file, the password has added quotes which cause authentication failures for anything using `ZITI_PWD` (i.e. `zitiLogin`)

This appears to be related to the `persistEnvironmentValues` function in the `ziti-cli-functions.sh` script.

Example taken from the .env file, notice the extra single quotes when the password should be simply `MyPass!`
```
if [[ "$ZITI_PWD" == "" ]]; then export ZITI_PWD="'MyPass!'"; else echo "NOT OVERRIDING: env var ZITI_PWD already set. using existing value"; fi
```

**Comment by plorenz** [comment]
Closed when edge issue with same number was merged. Reopening.

**Comment by gberl002** [comment]
Fixed by #1475
